### DISCUSSION TITLE
Custom subdomain HTTPS failed creation of Azure App Service Managed Certificate

### DISCUSSION THREAD

**Discussion by istvanamolnar** [vercel/community#7387] [Help] [answered]
### Summary

I am trying to register a custom domain for an Azure App Service app.
Registration of the domain was successful and the app is available as HTTP.
<img width="682" alt="image" src="https://github.com/vercel/community/assets/32988453/f73cfa19-0bb2-48bc-a5b4-5e4d5e4bea69">

I am unable to create any certificate for my domain on Azure, I am not even sure, which direction to look for the solution.
<img width="353" alt="image" src="https://github.com/vercel/community/assets/32988453/ea70eae7-0e93-46eb-86db-fb42246e2a46">
Validation succeeds (I added the required CN and TXT), Adding fails.

"Create App Service Managed Certificate and configure SSL binding
Failed to create App Service Managed Certificate for chat.easeme.eu due to error: Pending managed certificate failed: Certificate creation was rejected by CA for canonical name chat.easeme.eu:  If retrying does not help, please contact support for assistance."

Is it possible that the issue is due to my incorrect Vercel Settings?
I am on Hobby plan and got the following certs. on Vercel already:
"*.easeme.eu, easeme.eu"

Or should I keep looking for solution on Azure side?

Thanks a lot!
Istvan

### Steps to Reproduce

- Have a Vercel-hosted hobby project with a custom domain registered (e.g. easeme.eu)
- A running Azure App Service app
- Add binding to Custom Domain

**Comment by pawlean** [comment]
Hi @istvanamolnar!

Did you purchase the domain from Vercel? Or is it an external provider? This wasn't clear to me in your post.

**Reply by istvanamolnar** [reply]
I purchased from a 3rd party provider where the NS is set to Vercel

**Reply by pawlean** [reply]
I see! Any luck from Azure support?

**Reply by istvanamolnar** [reply, marked as answer]
I had to create a CAA record with value `0 issue "digicert.com"`

[a comment by an automated account is not shown]
